sale_order_team_from_product/models/sale_order.py

```
@@ -0,0 +1,95 @@
+    @api.depends("order_line.product_id.product_tmpl_id.sales_team_id")
+    def _compute_team_id(self) -> None:
+        """Extend the default team computation with product-based logic.
+
+        Base implementation computes the default Sales Team from the
+        salesperson, partner and company.
+
+        This override keeps the original behavior and,
+        if the feature is enabled, optionally replaces the result with the
+        team coming from the products on the order.
+        """
+        # pylint: disable=W8110
```

Pylint (4.0.3)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'W8110'

(W0012)